app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -432,6 +460,23 @@ const MeetingEditor = () => {
                     )}
                   </ConfirmModal>
                 )}
+                {isEditPage && canDelete && (
+                  <ConfirmModal
+                    title="Hente inn referat fra localstorage?"
+                    message={`Fant ${localStorage.getItem(`meeting-${meeting.id}-report`) !== null  ? 1 : 0} lagret backup for dette møtet. Dette vil overskrive det nåværende referatet lokalt.`}
```

Comment:
Why show the button if it does not find any local report? This will say "[...] found zero reports. Do you want to update?". Kind of weird

Reply:
This has been fixed and only shows if it finds something in localstorage.